code/modules/mapping/reader.dm

```diff
@@ -144,20 +144,23 @@
 	var/space_key = modelCache[SPACE_KEY]
 	var/list/bounds
 	src.bounds = bounds = list(1.#INF, 1.#INF, 1.#INF, -1.#INF, -1.#INF, -1.#INF)
+	var/did_expand = FALSE
```

Comment:
Could ya explain the purpose of this var?

Reply:
if the size of the map changes extools needs to know to update its grid